dashboards-observability/test/panels_constants.tsx

```diff
@@ -12,10 +12,23 @@ export const sampleSavedVisualization = {
   visualization: {
     id: 'oiuccXwBYVazWqOO1e06',
     name: 'Flight Count by Origin',
-    query:
-      'source=opensearch_dashboards_sample_data_flights | fields Carrier,FlightDelayMin | stats sum(FlightDelayMin) as delays by Carrier',
+    query: {
+      rawQuery:
```

Comment:
@SivaprasadAluri @mengweieric I don't see this structure change in the backend: https://github.com/opensearch-project/observability/blob/main/opensearch-observability/src/main/kotlin/org/opensearch/observability/model/SavedVisualization.kt#L24-L25

Are we yet to have a change in the backend for this?

Reply:
@ps48 rawQuery is not needed here, and its been reverted back to query